docs/distros/ubuntu.md

```diff
@@ -0,0 +1,38 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Ubuntu Launchpad PPA with `tezos-*` binaries
+
+If you are using Ubuntu you can use PPA in order to install `tezos-*` executables.
```

Comment:
Given that it provides this and more info perhaps it makes sense to link the baking guide here?

Reply:
Additional note: the section about "`tezos-baking` package" should take the suggestion above and I think also be moved on the top of the doc.